### PR TITLE
Ocd 1980 - Support many pending surveillance activities

### PR DESCRIPTION
**Pull request by tmy1313**

### CODE REVIEW

File: PENDING_PENDING_RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,7 +1,10 @@
 
-## Version 16.5.0
+## Version 17.0.0
 _Date TBD_
 
+### Major Change
+* When retrieving Pending Surveillances, the listing details will not longer be populated
```
Comment (andlar) [inline PENDING_PENDING_RELEASE_NOTES.md]: `not` -> `no`

File: chpl/chpl-service/src/main/java/gov/healthit/chpl/manager/impl/SurveillanceManagerImpl.java [hunk 2]
```diff
@@ -410,21 +410,7 @@ private Surveillance convertToDomain(final PendingSurveillanceEntity pr) {
         surv.setStartDate(pr.getStartDate());
         surv.setEndDate(pr.getEndDate());
         surv.setRandomizedSitesUsed(pr.getNumRandomizedSites());
-        if (pr.getCertifiedProduct() != null) {
```
Comment (andlar) [inline chpl/chpl-service/src/main/java/gov/healthit/chpl/manager/impl/SurveillanceManagerImpl.java]: There is some data about the Listing that's needed in the pending surveillance. Database ID, CHPL Product Number, maybe something else. Is that still getting populated somewhere?

File: chpl/chpl-service/src/main/java/gov/healthit/chpl/manager/impl/SurveillanceManagerImpl.java [hunk 3]
```diff
@@ -441,6 +427,11 @@ private Surveillance convertToDomain(final PendingSurveillanceEntity pr) {
                 reqType.setName(preq.getRequirementType());
                 req.setType(reqType);
 
+                CertifiedProduct cp = new CertifiedProduct();
+                cp.setId(pr.getCertifiedProductId());
+                cp.setChplProductNumber(pr.getCertifiedProductUniqueId());
```
Comment (andlar) [inline chpl/chpl-service/src/main/java/gov/healthit/chpl/manager/impl/SurveillanceManagerImpl.java]: Did you test these in the UI? I don't remember what all of the fields are that are required there, but the UI can't lose functionality based on not having the detailed information

Comment (tmy1313) [reply chpl/chpl-service/src/main/java/gov/healthit/chpl/manager/impl/SurveillanceManagerImpl.java]: Yes -  I tested the UI.  I uploaded a spreadheet, rejected and approved pending surveillances, and did some basic clicking around.  Seems to work w/o issues.

**Pull request merged**